src/client/components/carousel.js

```diff
@@ -23,12 +23,12 @@ export class Carousel extends Component {
       refresh: props.refresh || (() => { // get all docs from service by default
         const url = `/api/document`;
         const params = queryString.stringify({
-          status: ['published', 'submitted'].join(','),
```

Comment:
What is the current distinction between 'published' and 'submitted'?  It seems that the allowed statuses here should be the same as the allowed statuses for the download zip.

If there is no distinction between 'submitted' and 'published' anymore, then 'submitted' should be removed everywhere.

Reply:
The 'published' status is a shortcut to say that  the document meets some minimum standard: Contains at least one element and nothing is incomplete (e.g. un-grounded node).

Reply:
OK, then it sounds like the zip should only include 'published', whereas it includes 'submitted' right now (see https://github.com/PathwayCommons/factoid/pull/831).

Could that consistency be included in this PR?  This status issue has come up several times now, and it would be great to resolve it completely.

Reply:
OK will do